kinetic/distribution.yaml

```diff
@@ -7581,6 +7581,15 @@ repositories:
       type: git
       url: https://github.com/evocortex/optris_drivers.git
       version: kinetic-devel
+  orb_slam2_ros:
+    doc:
+      type: git
+      url: https://github.com/appliedAI-Initiative/orb_slam_2_ros.git
+    release:
```

Comment:
release sections require having run the bloom release tool or otherwise having created a git buildpackage compliat repository.

I think that you meant to create a `doc` and `source` entries.